packages/common/src/types.ts

```diff
@@ -62,7 +61,7 @@ export interface YogaInitialContext {
   extensions?: Record<string, any>
 }
 
-export interface RequestProcessContext<TContext, TRootValue> {
+export interface RequestProcessContext<TContext> {
```

Comment:
generic argument never used

Reply:
I think we can remove it. Maybe it can be breaking change for someone but maybe not since it makes no impact 🤔

Reply:
let's remove it in the next major with dropping support of Node.js 12 or we already dropped support?